docs/src/pages/components/view/demo.tsx

```diff
@@ -26,18 +27,26 @@ const propsToCode = (props: ViewProps) => {
   );
 };
 
+const defaultViewProps = {
+  as: 'div',
+  ariaLabel: 'View example',
+  width: '20rem',
+  height: '4rem',
+  color: 'var(--amplify-colors-blue-60)',
+  backgroundColor: 'var(--amplify-colors-white)',
+  boxShadow: '3px 3px 5px 6px var(--amplify-colors-neutral-60)',
+  padding: '1rem',
+  border: '1px solid var(--amplify-colors-black)',
+  borderRadius: '6px',
+};
+
 export const ViewDemo = () => {
-  const props = useViewProps({
-    as: 'div',
-    ariaLabel: 'View example',
-    width: '20rem',
-    height: '4rem',
-    color: 'var(--amplify-colors-blue-60)',
-    backgroundColor: 'var(--amplify-colors-white)',
-    boxShadow: '3px 3px 5px 6px var(--amplify-colors-neutral-60)',
-    padding: '1rem',
-    border: '1px solid var(--amplify-colors-black)',
-    borderRadius: '6px',
+  const props = useViewProps(
+    demoState.get(View.displayName) || defaultViewProps
+  );
+
+  React.useEffect(() => {
+    demoState.set(View.displayName, props);
   });
```

Comment:
Do you have the EsLint plugin installed? You should be getting a rules of hooks error here. I don't think we want to set the demoState on every render, only if props change:
```suggestion
  }, [props]);
```

Reply:
+1